poetry.lock

```diff
@@ -13,13 +13,13 @@ files = [
 
 [[package]]
 name = "accelerate"
-version = "0.34.2"
+version = "1.3.0"
```

Comment:
Please update and lock just the desired lib (litellm). Updating all dependencies is so dangerous
Use `poetry lock --no-update` to lock updated deps based on toml file